src/homepage/_posts/2020-04-06-james-3.5.0.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title:  "Apache James Server 3.5.0"
+date:   2020-04-06  15:16:30 +0200
+categories: james update
+---
+
+The Apache James developers are pleased to announce James server 3.5.0 release.
+
+Here are some points we worked on:
+
+### Added
+- Distributed task management for Guice cassandra-rabbitmq product. This enables several James servers to share a consistent view
+of tasks being currently executed.
+- JAMES-2563 Health check for ElasticSearch
+- JAMES-2904 Authentication and SSL support for Cassandra backend
+- JAMES-2904 Authentication and SSL support for ElasticSearch backend
+- JAMES-3066 Add "allowed From headers" webadmin endpoint
+- JAMES-3062 EventDeadLettersHealthCheck
+- JAMES-3058 WebAdmin offline task to correct mailbox inconsistencies on top of Cassandra products
+- JAMES-3105 WebAdmin offline task to recompute mailbox counters on top of Cassandra products
+- JAMES-3072 Webadmin endpoint to export mailbox backup
+
+### Changed
+- Multiple changes have been made to enhance ElasticSearch performance:
+  - Use of routing keys to collocate documents per mailbox
+  - Under some configuration, html was not extracted before document indexing
+  - Removed unnecessary fields from mailbox mapping
+  - Disable dynamic mapping thanks to a change of the header structure 
+  - Read related [upgrade instructions](upgrade-instructions.md)
+- JAMES-2855 Multiple library/plugin/docker images/build tool upgrades
+- By default the cassandra keyspace creation by James is now disabled by default. This allow to have credentials limited to a keyspace. It can be enabled by setting cassandra.keyspace.create=true in the cassandra.properties file.
```

Comment:
Do we have a nice documentation about how to deploy a keyspace for a new James instance?

Reply:
No

Reply:
Well, that's an unfair answer. The question is not about "who is gonna do it" but "is it an important piece of information".
We changed the way James is deployed and may have miss part of the documentation.
I think that without this doc, people will just fail and drop the product.
Either you agree or you disagree but please first express your opinion and then we'll figure out how we handle the change if it seems important.